config/cable.yml

```diff
@@ -0,0 +1,10 @@
+development:
+  adapter: async
+
+test:
+  adapter: test
+
+production:
+  adapter: redis
+  url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
```

Comment:
Did you chose database `1` intentionally?

Reply:
Generated by Rails as part of the upgrade, I believe. 🤷